test/RepositoryTest.php

```diff
@@ -65,6 +65,32 @@ public function testLoadVersions()
                     'type' => 'drupal-module'
                 ),
             ),
+            'drupal' => array(
+                '7.x-dev' => [
+                    'type' => 'drupal-core'
+                ],
+                '7.43.0' => [
+                    'type' => 'drupal-core'
+                ],
+                '8.0.x-dev' => [
+                    'type' => 'drupal-core'
```

Comment:
Drupal Packagist rewrites the package type to "drupal-core". Maybe we should keep "project"?